.github/workflows/build.yml

```diff
@@ -16,6 +16,9 @@ on:
 
 env:
   MAIN_CRATE: portable-rustls
+  # NOTE: USING this unstable cfg option to simplify running the existing CI jobs in this CI workflow
+  # TODO: test as many jobs as possible with default build configuration (using `portable_atomic_util::Arc`)
+  RUSTFLAGS: --cfg unstable_use_arc_from_stdlib
```

Comment:
Enabling an unstable cfg option in order to pass all tests in this workflow is a temporary workaround. I think it would be much better to update & adapt as many jobs as possible to test both with and without this unstable cfg option, in a similar fashion to how I updated this build workflow in experimental PR #52.

I think the updates in PR #52 are quite extensive, noticed that GitHub collapses the diff for this file in PR #52. I may want to defer this kind of workflow update for now.

Possible workaround may be to do this for each CI job as needed in this workflow, with QUICK WORKAROUND & TODO comments to explain the situation and what needs to be done at some point.